meetings/2023-06-21.md

```diff
@@ -0,0 +1,130 @@
+
+### Debriefing the screencast
+
+* SC: I'll see what I can do with the screencast. Useful even out of the context of this call.
+* HS: I think we should adapt the rules. The privacy concerns have evolved since COVID.
+* SC: the consent of the group is still necessary before any recording. 
+
+
+### Add WebID to Dependencies
+URL: https://github.com/solid/solid-wg-charter/pull/40
+
+* SC: Resolves https://github.com/solid/solid-wg-charter/issues/39 .
+>* SC: I suggest if no objections by 2023-06-21, we can merge with WebID CG Chair's go ahead on behalf of the group.
+>* HS: Ok, I sent [an email message out to WebID CG group](https://lists.w3.org/Archives/Public/public-webid/2023Jun/0001.html). Seems people were ok. But we can wait a week.
+* SC: How about now?
+  Note that the goal is not to turn WebID into a purely Solid thing. The WebID deliverable from the Solid WG would be usable in other contexts.
+* HS: I sent a link to the PR to the group, and I didn't see any objection.
+* SC: I followed the thread, I saw no objection either. But I'd like you as the chair to officially make that decision.
+* HS: Solid is actually using WebID, that's where it should be going.
+* SC: ACTION pchampin to merge following bblfish's approval in the PR
+
+
+### Update mission
+URL: https://github.com/solid/solid-wg-charter/issues/7
+
+* SC: This is editorial. PRs welcome.
+
+
+### Alternative solutions to container HATEOAS
+URL: https://github.com/solid/specification/issues/525
+
+* SC: Proposed by RG and eP.
+* RG: if you want to upload something in a container, then GET the container, it is not clear what you should retrieve: only a link to the uploaded content, or also a description of it?
+  Maybe a `Prefer` header could be used? eP has made a proposal in that sense.
+* SC: can you explain what you mean by "upload something"?
+* RG: imagine a container `/foo/`. Either you POST `bar` of put to `foo/bar`. What if you want to put in the container that is elsewhere?
+* SC: Can't link to a resource outside of the container path since client can't update the containment statements.
+* PA: Ran into this.. Re-read the LDP spec for that matter, it seems to me that LDP contains some guidance on Prefer header. I tested on CSS and NSS. It appears not implemented. LDP says MAY (or SHOULD?). Just implementing LDP may solve that problem. https://www.w3.org/TR/ldp/#prefer-parameters
+* eP: for me it is a broader issue. It could be a custom relation, not only `ldp:contains`. Related to server-managed statements, which the client can not fully control. Also occurs in Notifications where server should manage Subscription Service discovery. We need a longer meeting to dive into this problem. (discussion about the managed statement occurring in an auxiliary resource). Look at the issue that RG created, it also links to other long-discussed issues.
```

Comment:
```suggestion
* RG: Imagine a container `/foo/`. Either you `POST` to `bar` or `PUT` to `foo/bar`. What if you want to put something in the container that is elsewhere?
* SC: Can't link to a resource outside of the container path, since client can't update the containment statements.
* PA: Ran into this. Re-read the LDP spec for that matter. It seems to me that LDP contains some guidance on `Prefer` header. I tested on CSS and NSS. It appears not implemented. LDP says `MAY` (or `SHOULD`?). Just implementing LDP may solve that problem. https://www.w3.org/TR/ldp/#prefer-parameters
* eP: For me, it is a broader issue. It could be a custom relation, not only `ldp:contains`. Related to server-managed statements, which the client cannot fully control. Also occurs in Notifications where server should manage Subscription Service discovery. We need a longer meeting to dive into this problem. (discussion about the managed statement occurring in an auxiliary resource). Look at the issue that RG created; it also links to other long-discussed issues.
```